src/Akeneo/Connectivity/Connection/front/src/connect/components/ConnectedApp/OpenAppButton.tsx

```diff
@@ -0,0 +1,35 @@
+import {Button} from "akeneo-design-system";
+import {Translate, useTranslate} from '../../../shared/translate';
+import React, {FC} from "react";
+import {useRouter} from '../../../shared/router/use-router';
+import {useSecurity} from '../../../shared/security';
+import {ConnectedApp} from "@src/model/Apps/connected-app";
+import returnIconUrl from '../../../common/assets/icons/return.svg';
+
```

Comment:
Same remark as Willy here. Please check if it is fixed when using linter and if not create an octobug to add the rule in the linter.

```suggestion
import {Button} from 'akeneo-design-system';
import {Translate, useTranslate} from '../../../shared/translate';
import React, {FC} from 'react';
import {useRouter} from '../../../shared/router/use-router';
import {useSecurity} from '../../../shared/security';
import {ConnectedApp} from '@src/model/Apps/connected-app';
import returnIconUrl from '../../../common/assets/icons/return.svg';

```